extension/remotetapextension/main/main.go

```diff
@@ -0,0 +1,49 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Is there any reason for a main file for this?

Reply:
Maybe there was but not attached to it, deleting